cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -614,7 +614,10 @@ private String getClassPath(JBallerinaBackend jBallerinaBackend, Package current
             if (!module.testDocumentIds().isEmpty() || module.project().kind()
                     .equals(ProjectKind.SINGLE_FILE_PROJECT)) {
                 for (JarLibrary jarLibs : jarResolver.getJarFilePathsRequiredForTestExecution(module.moduleName())) {
-                    dependencies.add(jarLibs.path());
+                    if (jarLibs.path().toFile().getName().contains("testerina-runtime")
```

Comment:
We can use the provided APIs to get these jars.
i.e
`ProjectUtils.testDependencies()`
`jBallerinaBackend.runtimeLibrary().path()`